shared/chat/conversation/messages/wrapper/exploding-height-retainer/index.desktop.js

```diff
@@ -24,9 +23,17 @@ type State = {
   height: number,
 }
 class ExplodingHeightRetainer extends React.PureComponent<Props, State> {
+  boxRef: React.ElementRef<any>
```

Comment:
these should start w/ underscore, why use any

Reply:
ok